lib/Wikia/src/Service/User/Auth/MediaWikiAuthService.php

```diff
@@ -2,8 +2,66 @@
 
 namespace Wikia\Service\User\Auth;
 
+use Wikia\Service\Helios\ClientException;
+use Wikia\Service\Helios\HeliosClient;
+use Wikia\Logger\Loggable;
+
 class MediaWikiAuthService implements AuthService {
+	use Loggable;
+
+	private $heliosClient;
+
+	/**
+	 * @Inject({Wikia\Service\Helios\HeliosClient::class})
+	 */
+	public function __construct( HeliosClient $client ) {
+		$this->heliosClient = $client;
+	}
+
+	/**
+	 * @param HeliosClient $client
+	 */
+	public function setHeliosClient( HeliosClient $client ) {
+		$this->heliosClient = $client;
+	}
+
+	public function authenticate( $username, $password ): AuthResult {
+		$this->info( 'HELIOS_LOGIN authenticate', [ 'method' => __METHOD__, 'username' => $username ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "username" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "method" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)